pkg/types/core/relayerset.go

```diff
@@ -27,7 +27,10 @@ type RelayArgs struct {
 	MercuryCredentials *types.MercuryCredentials
 }
 
+//go:generate mockery --quiet --name Relayer --output ./mocks/ --case=underscore
```

Comment:
Please use the `.mockery.yaml` instead of `go:generate`

Reply:
This interface is already in .mockery.yaml